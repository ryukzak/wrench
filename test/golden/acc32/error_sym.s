    .data

cc:              .word  0xCCCCCCCC
ff:              .word  0xFFFFFFFF
o:               .word  'o'
output_addr:     .word  0x84               ; Output address where the result should be stored

    .text

_start:

    load_addr    cc
    store_ind    output_addr

    load_addr    ff
    store_ind    output_addr

    load_addr    o
    store_ind    output_addr

    halt
