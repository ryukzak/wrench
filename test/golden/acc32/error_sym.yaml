name: error symbols output
limit: 400
memory_size: 0x1000
input_streams:
  0x80: []
  0x84: []
reports:
  - name: Step-by-step
    slice: all
    view: |
      {pc}:	{instruction}	{pc:label} {Acc}
  - name: Result
    slice: last
    view: |
      numio[0x80]: {io:0x80:dec}
      numio[0x84]: {io:0x84:dec}
      symio[0x80]: {io:0x80:sym}
      symio[0x84]: {io:0x84:sym}
      {memory:0:11}
    assert: |
      numio[0x80]: [] >>> []
      numio[0x84]: [] >>> [-858993460,-1,111]
      symio[0x80]: "" >>> ""
      symio[0x84]: "" >>> "##o"
      mem[0..11]: 	cc cc cc cc ff ff ff ff 6f 00 00 00
